pkg/operator/controller/controller.go

```diff
@@ -775,6 +776,9 @@ func (r *ReconcileVMImportConfig) getOperatorArgs(cr *v2vv1.VMImportConfig) *Ope
 		if cr.Spec.ImagePullPolicy != "" {
 			result.PullPolicy = string(cr.Spec.ImagePullPolicy)
 		}
+		if &cr.Spec.Infra != nil {
```

Comment:
If you define Infra 
`NodePlacement `json:"infra,omitempty"``
as
`Infra *NodePlacement `json:"infra,omitempty"``

you could just check 
```
if cr.Spec.Infra != nil {
...
```

without need to check the address with &. This seems to complete the API notation of having an optional element.
Is there any specific reason not to define the NodePlacement on the VMImportConfig as *NodePlacement ?

Reply:
Wtih you suggestion compiler complains:
> pkg/operator/controller/controller.go:779:20: cannot convert nil to type "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1beta1".NodePlacement

Reply:
The only reason is consistency with CDI.

Reply:
I'm referring to the guidelines on https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
> The only reason is consistency with CDI.

okay  - so we can stick to it.